editor/src/core/workers/parser-printer/parser-printer-parsing.ts

```diff
@@ -1915,6 +1911,45 @@ function buildHighlightBounds(
   }
 }
 
+function buildHighlightBoundsForExpressionsAndText(
+  sourceFile: TS.SourceFile,
+  expressions: Array<ExpressionAndText<TS.Node>>,
+  uid: string,
+): HighlightBounds {
+  // Default to using this first.
+  const baseHighlightBounds = buildHighlightBounds(
+    sourceFile,
+    mapDropNulls((e) => e.expression, expressions),
+    uid,
+  )
+  if (baseHighlightBounds == null) {
+    // Find the bounds of the bounds.
+    let lowestStart: number | null = null
+    let highestEnd: number | null = null
+    for (const expression of expressions) {
+      lowestStart = Math.min(lowestStart ?? expression.startPos, expression.startPos)
+      highestEnd = Math.min(highestEnd ?? expression.endPos, expression.endPos)
```

Comment:
shouldn't this be Math.max to get the "highest" in highestEnd?

Reply:
It definitely should, copy-paste considered dangerous...Fixed in d71a88628.